3-make_gene_matrix.R

```diff
@@ -67,6 +63,9 @@ option_list <- list(
 # Parse options
 opt <- parse_args(OptionParser(option_list = option_list))
 
+opt$dir <- "patel_data/salmon_quants"
```

Comment:
I don't think you want to hardcode these? Is this leftover from development?

Reply:
Yes. Ignore that. I'll delete it.